src/viewer.ts

```diff
@@ -524,6 +527,29 @@ export class SkinViewer {
 		}
 	}
 
+	//james090500
+	private frameDelay = 100;
```

Comment:
Should this be controllable by the consumer?

Reply:
I believe this has been added